bindings/python/test/test_mongocrypt.py

```diff
@@ -123,16 +127,16 @@ def test_mongocrypt_options_validation(self):
                 {'aws': {'accessKeyId': 'foo'}},
                 {'aws': {'secretAccessKey': 'foo'}}]:
             with self.assertRaisesRegex(
-                    ValueError, "kms_providers\['aws'\] must contain "
+                    ValueError, r"kms_providers\['aws'\] must contain "
```

Comment:
These changes address warnings seen while running the tests:

```python
test/test_mongocrypt.py:139
  /Users/steve.silvester/workspace/libmongocrypt/bindings/python/test/test_mongocrypt.py:139: DeprecationWarning: invalid escape sequence '\('
    "instance of bytes or str \(unicode in Python 2\)"):
```

Reply:
Noted.